src/clusterfuzz/_internal/platforms/android/fetch_artifact.py

```diff
@@ -148,10 +156,13 @@ def get_latest_artifact_info(branch, target, signed=False):
       successful=True,
       maxResults=1,
       signed=signed)
+  request_str = '{}, {}, {}, {}'.format(request.uri, request.method,
```

Comment:
Same comment here re using `', '.join` instead. ,

Reply:
Same reason as above.

Reply:
Just use an f-string instead of calling format.